aws/ecs-service/main.tf

```diff
@@ -79,6 +79,28 @@ resource "aws_ecs_task_definition" "this" {
   tags = local.tags
 }
 
+resource "aws_ecs_task_definition" "job" {
+  count                    = var.create_job_task ? 1 : 0
```

Comment:
We're missing a `variable` in `aws/ecs-service/variables.tf` for the new var used here.

```hcl
variable "create_job_task" {
  description = "Whether to create a job task. Defaults to `false`."
  type        = bool
  default     = false
}

Reply:
Sorry, added it now